src/sas/qtgui/Perspectives/Fitting/FittingWidget.py

```diff
@@ -1131,13 +1131,14 @@ def getConstraintsForFitting(self):
             msg = "The current fit contains constraints relying on other fit pages.\n"
             msg += "Parameters with those constraints are:\n" +\
                 '\n'.join([cons[0] for cons in multi_constraints])
-            msg += "\n\nWould you like to remove these constraints or cancel fitting?"
+            msg += "\n\nWould you like to deactivate these constraints or " \
+                   "cancel fitting?"
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) recommends parentheses instead of line continuation character for breaking expressions over multiple lines..